src/Tec/Plugin.php

```diff
@@ -193,25 +186,20 @@ function reschedule_trash_or_del_event_cron( $cron ) {
 		 * - fifteendays
 		 * - monthly
 		 */
+		$frequency_struct = explode( '|', $time );
+		$feq              = $frequency_struct[0];
+		$interval         = $frequency_struct[1] ?? 'month';
 
-		// For 1 minute and 15 minutes
-		if ( $time < 60 ) {
-			$frequency = 'tribe-every15mins';
+		// For 15 minutes (and other minutes)
+		if ( str_starts_with( $interval, 'minute' ) ) {
+			return 'tribe-every15mins';
 		}
-		// For 1 hour and 12 hours
-		elseif ( $time < 1440 ) {
-			$frequency = 'hourly';
-		}
-		// For 1 day and longer
-		else {
-			$frequency = 'daily';
+		// For 1 hour and 12 hours (and other hours)
+		elseif ( str_starts_with( $interval, 'hour' ) ) {
+			return 'hourly';
 		}
 
-		$scheduled = wp_next_scheduled( $cron );
-
-		if ( $scheduled && $frequency !== wp_get_schedule( $cron ) ) {
-			wp_unschedule_event( $scheduled, $cron );
-			wp_schedule_event( time(), $frequency, $cron );
-		}
```

Comment:
Why are we no longer scheduling events ?

Reply:
The trash cron is set up automatically by TEC.
We have a filter that allows us to change the frequency of the trash cron, and that filter is being used in the extension.
(Line 96)